### PR TITLE
[16.0][MIG] stock_available_mrp: Migration to 16.0

### PR DESCRIPTION
**Pull request by khoivo1601**

### CODE REVIEW

File: stock_available_mrp/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2014 Numérigraphe SARL, Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Consider the production potential is available to promise",
+    "version": "16.0.1.0.1",
```
Comment (rousseldenis) [inline stock_available_mrp/__manifest__.py]: Please reset to `16.0.1.0.0`

**Review by rousseldenis — COMMENTED** [review]
@khoivo1601 Thanks for this. LGTM. Minor comment

File: stock_available_mrp/__manifest__.py (hunk 1 above)
Comment (khoivo1601) [reply stock_available_mrp/__manifest__.py]: ah my bad, i updated `"version": "16.0.1.0.0"`

**Comment by khoivo1601** [comment]
hi @rousseldenis, thanks for your review. This PR has been updated !

**Review by leemannd — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**